tests/projects.spec.ts

```diff
@@ -10,30 +10,55 @@ async function forceLoadImages(page: Page) {
 	await page.getByRole('contentinfo').scrollIntoViewIfNeeded();
 	await page.locator('#event-1').scrollIntoViewIfNeeded();
 	await page.waitForTimeout(500);
-	await page.getByRole('list').getByRole('heading', { name: 'Silvester' }).scrollIntoViewIfNeeded();
+	await page
+		.getByRole('list')
+		.getByRole('heading', { name: 'Silvester' })
+		.or(page.getByRole('list').getByRole('heading', { name: 'Mitgliederversammlung' }))
+		.scrollIntoViewIfNeeded();
 	await page.waitForTimeout(500);
 }
 
+async function expectForAllSelected(page: Page) {
+	for (const e of await page.getByRole('list').getByText('Nur Mitglieder').all()) {
+		await expect(e).not.toBeVisible();
+	}
+	for (const e of await page.getByRole('list').getByText('Für Alle').all()) {
+		await expect(e).toBeVisible();
+	}
+}
+
+async function expectOnlyMemebersSelected(page: Page) {
+	for (const e of await page.getByRole('list').getByText('Nur Mitglieder').all()) {
+		await expect(e).toBeVisible();
+	}
+	for (const e of await page.getByRole('list').getByText('Für Alle').all()) {
+		await expect(e).not.toBeVisible();
+	}
+}
+
 test('Projects page screenshot', async ({ page }) => {
 	await forceLoadImages(page);
 
 	await expect(page).toHaveScreenshot({ fullPage: true });
 });
 
-test('Filter members_only', async ({ page }) => {
+test('Select members_only', async ({ page }) => {
 	await page.getByLabel('Nur Mitglieder').click();
-	await expect(page.getByRole('list').getByText('Nur Mitglieder')).not.toBeVisible();
+	await expectOnlyMemebersSelected(page);
 
 	await forceLoadImages(page);
 
 	await expect(page).toHaveScreenshot({ fullPage: true });
 });
 
-test('Filter for_all', async ({ page }) => {
+test('Reselect for_all', async ({ page }) => {
```

Comment:
I think I would keep the test name as is. That way we won't have to rename the screenshots.

Reply:
Yeah, I just thought I had to update the screenshots anyway.